imod/tests/test_mf6/test_mf6_hfb_adapter.py

```diff
@@ -118,5 +118,13 @@ def test_hfb_writing_one_layer__unstructured(barrier, tmp_path):
     hfb.write("hfb", None, write_context)
 
     # Assert
-    data = np.loadtxt(tmp_path / "hfb" / "hfb.dat")
+    hfb_bin_path = tmp_path / "hfb" / "hfb.bin"
+    hfb_txt_path = tmp_path / "hfb" / "hfb.dat"
+
+    # Test if not written to binary file
```

Comment:
I would remove this comment and the one on line 126. I think it is clear from the code what it does.
If you do want to add that info you could consider renaming the test name to something like `test_hfb_writing_one_layer_non_binary_format`

Reply:
Done